.circleci/config.yml

```diff
@@ -30,14 +30,22 @@ aliases:
 executors:
   node-executor:
     docker:
-      - image: circleci/node:latest
+      - image: cimg/node:current
     working_directory: ~/babel
   # e2e-vue-cli test requires chromium
   node-browsers-executor:
     docker:
-      - image: circleci/node:latest-browsers
+      - image: cimg/node:current-browsers
```

Comment:
The e2e test error is related: it seems that `cimg/node:current-browsers` is lack of some libraries.